includes/subscriptions/class-wc-payments-subscriptions-event-handler.php

```diff
@@ -123,8 +122,10 @@ public function handle_invoice_paid( array $body ) {
 			$order->payment_complete();
 		}
 
-		// Add the payment intent data to the order.
-		$this->invoice_service->get_and_attach_intent_info_to_order( $order, $wcpay_intent_id );
+		if ( isset( $event_object['payment_intent'] ) ) {
```

Comment:
Because the first subscription invoice is paid out of band, when we receive the paid invoice webhook, it no longer has a payment intent ID. This was causing the error in WC Pay logs that I mentioned on the PR description.

I've tested invoice paid webhooks for renewal orders and in those cases it is present and passes this condition correctly.